ComplexJobTrigger.ERROR_0001_InvalidCronExpression=Invalid cron expression.
ActionAdapterQuartzJob.ERROR_0001_REQUIRED_PARAM_MISSING=Property "{0}" or "{1}" must be set in the job data map
ActionAdapterQuartzJob.ERROR_0002_FAILED_TO_CREATE_ACTION=Failed to create an instance of action "{0}": {1}
ActionAdapterQuartzJob.ERROR_0003_ACTION_WRONG_TYPE=class {0} must be an instance of "{1}"
ActionAdapterQuartzJob.ERROR_0004_ACTION_FAILED=Action "{0}" failed to run as a quartz job
ActionAdapterQuartzJob.WARN_0001_SKIP_REMOVING_OUTPUT_FILE=File written by XActions must be cleaned up by external means: {0}
ActionAdapterQuartzJob.WARN_0002_NO_STATUS=Status for action "{0}" is not available; the action may have been run \
  remotely: {1}

QuartzJobKey.ERROR_0000=Cannot generate job key, jobName is missing
QuartzJobKey.ERROR_0001=Cannot generate job key, username is missing
QuartzJobKey.ERROR_0002=Missing data in jobId "{0}".
QuartzScheduler.ERROR_0001_FAILED_TO_SCHEDULE_JOB=Quartz failed to schedule job "{0}"
QuartzScheduler.ERROR_0002_TRIGGER_WRONG_TYPE=Trigger is an illegal type
QuartzScheduler.ERROR_0003_ACTION_IS_NULL=Action cannot be null
QuartzScheduler.ERROR_0004_FAILED_TO_LIST_JOBS=Quartz failed to list jobs.
QuartzScheduler.ERROR_0005_FAILED_TO_PAUSE_JOB=Quartz failed to pause job.
QuartzScheduler.ERROR_0005_FAILED_TO_RESUME_JOB=Quartz failed to resume job.
QuartzScheduler.ERROR_0006_FAILED_TO_GET_SCHEDULER_STATUS=Unable to get Quartz scheduler status.
QuartzScheduler.ERROR_0007_FAILED_TO_GET_JOB=Failed to get job "{0}"
QuartzScheduler.ERROR_0008_SCHEDULING_IS_NOT_ALLOWED=You don't have scheduling permissions for this file.\nContact your administrator for assistance.
QuartzScheduler.ERROR_0009_SCHEDULING_IS_NOT_ALLOWED_AFTER_CHANGE=Due to user/role changes, job "{0}" is not able to be executed by "{1}"
QuartzScheduler.ERROR_0010_UNAVAILABLE_OUTPUT_LOCATION=Unavailable output location, "{0}", for job "{1}" of user "{2}". Location does not exist, is not a folder, the user lacks read/write access to it, or location is of an invalid or currently inaccessible VFS connection.
QuartzScheduler.ERROR_0011_UNAVAILABLE_OUTPUT_LOCATION_ERROR=Unavailable output location, "{0}", for job "{1}" of user "{2}". Will try fallback locations.
QuartzScheduler.ERROR_0012_UNAVAILABLE_OUTPUT_LOCATION_FALLBACK=Unavailable fallback output location, "{0}", for job "{1}" of user "{2}". Location does not exist, is not a folder, the user lacks read/write access to it, or location is of an invalid or currently inaccessible VFS connection.
QuartzScheduler.ERROR_0013_UNAVAILABLE_OUTPUT_LOCATION_FALLBACK_ERROR=Unavailable fallback output location, "{0}", for job "{1}" of user "{2}".
QuartzScheduler.ERROR_0014_FOUND_AVAILABLE_OUTPUT_LOCATION_FALLBACK=Found available fallback output location, "{0}", for job "{1}" of user "{2}".
QuartzScheduler.ERROR_0015_NO_AVAILABLE_OUTPUT_LOCATION_FALLBACK=Could not find an available fallback output location for job "{0}" of user "{1}".
QuartzScheduler.ERROR_0016_UNAVAILABLE_OUTPUT_LOCATION=User lacks privileges to the output location.
EmbeddedQuartzSystemListener.ERROR_0001_Scheduler_Not_Initialized=Scheduler was not properly initialized at startup
EmbeddedQuartzSystemListener.ERROR_0004_LOAD_PROPERTIES_FROM_CLASSPATH=Loading quartz.properties from classpath failed.
EmbeddedQuartzSystemListener.ERROR_0005_UNABLE_TO_INSTANTIATE_OBJECT=Unable to instantiate object
EmbeddedQuartzSystemListener.ERROR_0006_UNABLE_TO_GET_DATASOURCE=Unable to get datasource object
EmbeddedQuartzSystemListener.ERROR_0007_SQLERROR=SQL Error creating Quartz tables
EmbeddedQuartzSystemListener.ERROR_0008_UNABLE_TO_FIND_INIT_SCRIPT=Cannot find Quartz initialization script system/quartz/quartzinit.sql
JobParamsAdapter.ERROR_0001=Type {0} not supported by {1}

schedulerEmailFromName=Pentaho Scheduler
PentahoBlockoutManager.ERROR_0001_WRONG_BLOCKER_TYPE=Attempted to add a IBlockOutTrigger object that is not an instance of Trigger
PentahoBlockoutManager.ERROR_0002_CANT_CREATE_BLOCKOUT=Unable to locate block out with name: "{0}"
PentahoBlockoutManager.ERROR_0003_CANT_PARSE_RECURRENCE_INTERVAL=Can not parse a valid recurrence interval from the provided Trigger

Unknown=Unknown

ActionInvoker.INFO_0001_RUNNING_IN_BG_LOCALLY=Running action "{0}" in background locally: {1}

ActionInvoker.WARN_0001_SKIP_REMOVING_OUTPUT_FILE=File written by XActions must be cleaned up by external means: {0}

ActionInvoker.ERROR_0004_ACTION_FAILED=Action "{0}" failed to execute
ActionInvoker.ERROR_0005_ACTION_NULL=Action is null, cannot invoke
ActionInvoker.ERROR_0006_ACTION_NULL=Action "{0}" is not supported
ActionInvoker.ERROR_0008_MAP_NULL_CANT_RETURN_SP=Map is null, cannot return stream provider
ActionInvoker.ERROR_0010_CANNOT_GET_REPO_FILE=Cannot get repository file "{0}": {1}

PentahoPlatformExporter.INFO_START_EXPORT_SCHEDULE=*********************************  [ Start: Backup Schedule ] *******************************
PentahoPlatformExporter.INFO_END_EXPORT_SCHEDULE=*********************************  [ End: Backup Schedule ] *******************************
PentahoPlatformExporter.INFO_COUNT_SCHEDULE_TO_EXPORT=Found [ {0} ] schedule(s) to backup
PentahoPlatformExporter.INFO_SUCCESSFUL_SCHEDULE_EXPORT_COUNT=Successfully perform backup of [ {0} ] out of [ {1} ] schedule(s)
PentahoPlatformExporter.ERROR_SCHEDULE_EXPORT=Error performing backup of schedule [ {0} ] cause [ {0} ]
